### PR TITLE
First implementation of interaction tests with Axe Accessibility scanning

### PR DESCRIPTION
**Pull request by marcelwgn**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Adds automated Axe accessibility scanning to catch Accessibility errors faster

**Details of Changes**
Add details of changes here.
- Add interaction tests project with Axe tests and necessary infrastructure
- Fix one a11y error
- Add automation IDs to elements for interaction testing

**Validation**
How did you test these changes?
- [X] Built and ran the app
- [X] Tested the changes for accessibility
- [X] Add documentation

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

**Comment by marcelwgn** [comment]
@yair100 Should we create a issue to track the other things that we also should do (rename folder, add runs to CI,...)?

**Comment by yair100** [comment]
> @yair100 Should we create a issue to track the other things that we also should do (rename folder, add runs to CI,...)?

Good idea, please tag me on them.

**Comment by yaichenbaum** [comment]
@chingucoding will this run as part of the CI or only locally?

**Comment by chingucoding** [comment]
> @chingucoding will this run as part of the CI or only locally?

Right now, only locally. I would add this to a CI in a different PR if that's fine.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -47,6 +47,11 @@ to group your changes into a small number of commits which we can review one at
 When completing a pull request, we will generally squash your changes into a single commit. Please
 let us know if your pull request needs to be merged as separate commits.
 
+## Running interaction tests
```
Comment (yaichenbaum) [inline .github/CONTRIBUTING.md:50]:
```suggestion
## Running interaction tests (optional)
```

**Review by yair100 — APPROVED** [review]

**Comment by marcelwgn** [comment]
Should we merge this than or would you like me to do more changes @yair100 ? :)

**Comment by yair100** [comment]
> Should we merge this than or would you like me to do more changes @yair100 ? :)

It looks good to me, I just want Gave to look it over before I merge it.

**Pull request merged**